SampleApp/Podfile

```diff
@@ -1,6 +1,4 @@
 platform :ios, '12.0'
-inhibit_all_warnings!
-use_frameworks!
```

Comment:
This fixes a weird Cocoapods thing I keep running into locally, where it's not putting the re-built framework into the simulator, so it crashes.